app/src/main/java/com/example/samplegallery/PhotoBlowupFragment.java

```diff
@@ -86,7 +86,7 @@ public void onResponse(JSONObject response) {
                         try {
                             sizes = response.getJSONObject("sizes").getJSONArray("size");
                             photoUrl = sizes
-                                    .getJSONObject(sizes.length() - 1)
+                                    .getJSONObject(sizes.length() - 6)
```

Comment:
This isn't really safe! What happens when sizes.length() is less than 6? 

Also, this seems to be loading a tiny image, isn't it?